modules/reference/pages/rpk/rpk-topic/rpk-topic-consume.adoc

```diff
@@ -10,293 +10,317 @@ The default output format `--format json` is a special format that outputs each
+value is a big-endian uint32, `%v` will print the raw four bytes, while
+`%v{unpack[>I]}` will print the number in as ASCII. If unpacking exhausts the
 input before something is unpacked fully, an error message is appended to the
 output.
 
 === Headers
 
 Headers are formatted with percent encoding inside of the modifier:
 
-[.no-copy]
-----
-    %h{ %k=%v{hex} }
-----
+```
+%h{%k=%v{hex}}
+```
 
 will print all headers with a space before the key and after the value, an
 equals sign between the key and value, and with the value hex encoded. Header
 formatting actually just parses the internal format as a record format, so all
-of the above rules about %K, %V, text, and numbers apply.
+of the above rules about `%K`, `%V`, text, and numbers apply.
 
 === Examples
 
-* A key and value, separated by a space and ending in newline:
-`-f '%k %v\n'`
-* A key length as four big endian bytes, and the key as hex:
-`+-f '%K\{big32}%k\{hex}'+`
-* A little endian uint32 and a string unpacked from a value:
-`-f '%v\{unpack[is$]}'`
+A key and value, separated by a space and ending in newline:
+
+```
+-f '%k %v\n'
+```
+
+A key length as four big endian bytes, and the key as hex:
+
+```
+-f '%K{big32}%k{hex}'
+```
+
+A little endian uint32 and a string unpacked from a value:
+
+```
+-f '%v{unpack[is$]}'
+```
 
 === Offsets
 
 The `--offset` flag allows for specifying where to begin consuming, and
-optionally, where to stop consuming. The literal words "start" and "end"
+optionally, where to stop consuming. The literal words `start` and `end`
 specify consuming from the start and the end.
 
-[.no-copy]
-----
-    start     consume from the beginning
-    end       consume from the end
-    :end      consume until the current end
-    +oo       consume oo after the current start offset
-    -oo       consume oo before the current end offset
-    oo        consume after an exact offset
-    oo:       alias for oo
-    :oo       consume until an exact offset
-    o1:o2     consume from exact offset o1 until exact offset o2
-    @t        consume starting from a given timestamp
-    @t:       alias for @t
-    @:t       consume until a given timestamp
-    @t1:t2    consume from timestamp t1 until timestamp t2
-----
+[cols=",",]
+|===
+|Offset |Description
+|`start` |Consume from the beginning.
+|`end` |Consume from the end.
+|`:end` |Consume until the current end.
+|`+oo` |Consume oo after the current start offset.
+|`-oo` |Consume oo before the current end offset.
+|`oo` |Consume after an exact offset.
+|`oo:` |Alias for oo.
+|`:oo` |Consume until an exact offset.
+|`o1:o2` |Consume from exact offset o1 until exact offset o2.
+|`@t` |Consume starting from a given timestamp.
+|`@t:` |alias for @t.
+|`@:t` |Consume until a given timestamp.
+|`@t1:t2` |Consume from timestamp t1 until timestamp t2.
+|===
 
 There are a few options for timestamps, with each option being evaluated
 until one succeeds:
```

Comment:
Suggest: Each timestamp option is evaluated until one succeeds.

Reply:
updated